src/button.tsx

```diff
@@ -16,7 +16,7 @@ import type {DataAttributes, RendersElement, RendersNullableElement, TrackingEve
 import type {Location} from 'history';
 import type {Theme} from './theme';
 
-export const BUTTON_MIN_WIDTH = 136;
+export const BUTTON_MIN_WIDTH = 104;
```

Comment:
Specs say 136
![image](https://user-images.githubusercontent.com/6722153/185189435-ad72e2b2-7fa6-46dc-900d-1299b9296745.png)

now the buttonPrimary has weird min-width
https://ibit.ly/2oW_
![image](https://user-images.githubusercontent.com/6722153/185189591-e1285e41-8bb5-41d5-9075-1b01eab6fcc3.png)

(before)
https://ibit.ly/vY8v
![image](https://user-images.githubusercontent.com/6722153/185189682-51ab5294-132c-4077-992d-9e2270818f4d.png)

Reply:
My error reading the specs, fixed primary min-width